### DISCUSSION TITLE
Docker completions not found

### DISCUSSION THREAD

**Discussion by onlinejudge95** [zdharma-continuum/zinit#669] [Q&A]
I installed docker in my fresh install of ubuntu 24.04 and after following the post installation step of removing sudo. I added the zinit snippet. Whenever I start my shell the following failure happens
```
tee: /home/onlinejudge95/.cache/zinit/completions/_docker: No such file or directory
```
Do I need to manually create the dir?

**Comment by softwareengineer-imerjr** [comment]
It is happening with me too. I'm trying to use docker completions directly from Oh My Zsh, but a'int got success with it...
